MANIFEST.in

```diff
@@ -15,18 +15,12 @@ include chroma-manager.py
 include chroma-manager.conf.template
 include mime.types
 include agent-bootstrap-script.template
-include chroma_core/fixtures/*
-include chroma_core/bin/*.py
-include chroma_core/bin/storage-daemon
-include chroma_core/bin/chroma-config
-include tests/sample_data/*
+include chroma_core/fixtures/*.json
+include tests/sample_data/*.json
 include tests/integration/run_tests
 include tests/integration/*/*.json
 include tests/integration/core/clear_ha_el?.sh
 include storage_server.repo
 include *.profile
 graft licenses
 graft polymorphic
-global-exclude .*.swp
-global-exclude local_settings.py*
```

Comment:
I think we may still want this one, as someone may have a local_settings.py override in their tree.